api/jsonrpc/eth_endpoint.go

```diff
@@ -117,12 +119,13 @@ func (e *Eth) GetTransactionReceipt(hash string) (interface{}, error) {
 	return nil, fmt.Errorf("transaction not found")
 }
 
-// CurrentBlock returns current block number
+// GetBalance returns the account's balance
 func (e *Eth) GetBalance(address string, number string) (interface{}, error) {
```

Comment:
GetBalance takes a BlockNumber reference (number or "latest").

Reply:
Clarified it in the comment 👍